esp/src/eclwatch/ESPWorkunit.js

```diff
@@ -312,7 +312,13 @@ define([
             WsWorkunits.WUUpdate({
                 request: request,
                 load: function (response) {
-                    context.updateData(response.WUUpdateResponse.Workunit);
+                    if (response.Exceptions) {
+                        arrayUtil.forEach(response.Exceptions.Exception, function (item, idx) {
+                            context.updateData(response.Exceptions.Exception[idx].Message);
```

Comment:
So you passing in a "string" to context.updateData?  I don't think it will like that.  And I don't thing you need to pass anything into updateData as per above.